models/stg_shopify__abandoned_checkout.sql

```diff
@@ -45,8 +45,8 @@ final as (
         billing_address_zip,
         buyer_accepts_marketing as has_buyer_accepted_marketing,
         cart_token,
-        cast(closed_at as {{ dbt.type_timestamp() }}) as closed_at,
-        cast(created_at as {{ dbt.type_timestamp() }}) as created_at,
+        {{ dbt_date.convert_timezone(column='cast(created_at as ' ~ dbt.type_timestamp() ~ ')', target_tz=var('shopify_timezone', "UTC"), source_tz="UTC") }} as created_at,
+        {{ dbt_date.convert_timezone(column='cast(closed_at as ' ~ dbt.type_timestamp() ~ ')', target_tz=var('shopify_timezone', "UTC"), source_tz="UTC") }} as closed_at,
```

Comment:
Are these additional quotes due to a database issue? Maybe because the `calogica` package doesn't work with Redshift?

Reply:
nah the quotes are just for any bits that are SQL and not jinja